scripts/deploy-federation.sh

```diff
@@ -215,3 +219,17 @@ CONTEXT="$(kubectl config current-context)"
 for c in ${JOIN_CLUSTERS}; do
   ./bin/kubefed2 join "${c}" --host-cluster-context "${CONTEXT}" --add-to-registry --v=2 ${KF_NS_ARGS}
 done
+
+if [ "`uname`" == 'Darwin' ];then
+
+    # We need to fix cluster ip addr in cluster-registry for mac os.
+    # Assume all context was contained in current kubeconfig.
```

Comment:
my kubeconfig has a few unrelated clusters...

Reply:
`for c in 'kubectl config get-contexts -o name';`   change to --->  `${JOIN_CLUSTERS}  `.  Is this ok with you?